src/pages/Trainee/TraineeList.jsx

```diff
@@ -2,23 +2,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import * as moment from 'moment';
+import {graphql} from '@apollo/client/react/hoc';
+import { flowRight as compose } from 'lodash';
 import { Button, withStyles } from '@material-ui/core';
 import EditIcon from '@material-ui/icons/Edit';
 import DeleteIcon from '@material-ui/icons/Delete';
 import { AddDialog, EditDialog, DeleteDialog } from './Component/index';
 import { TableComponent } from '../../components';
 import callApi from '../../libs/utils/api';
-
+import { trainees } from './Data/trainee';
```

Comment:
Why we need this anymore? This is static data now data should be fetched from API.